assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -334,15 +370,45 @@ export default function StoryListView({
                   )}
                 </TitleTableCellContainer>
               </TableCell>
-              <TableCell>{story.author || '—'}</TableCell>
-              <TableCell>{getRelativeDisplayDate(story.created)}</TableCell>
-              <TableCell>{getRelativeDisplayDate(story.modified)}</TableCell>
+              <TableCell>
+                <Text
+                  as="span"
+                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                >
+                  {story.author || '—'}
+                </Text>
+              </TableCell>
+              <TableCell>
+                <Text
+                  as="span"
+                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                >
+                  {getRelativeDisplayDate(story.created)}
```

Comment:
let's use `story.created_gmt` here so we can avoid situations where it appears the date is in the future - date structure changed fairly recently, this was probably just missed when that happened. I'm referencing storyGridView line 179.

![Screen Shot 2021-02-25 at 2 22 04 PM](https://user-images.githubusercontent.com/10720454/109219510-ef526c00-7774-11eb-8dcd-ce3b10e4f783.png)

```suggestion
                  {getRelativeDisplayDate(story.created_gmt)}
```